content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -68,19 +68,20 @@ interface
 
 ### Recovering the cluster configuration into the first node
 
-From the rladmin CLI prompt, run the cluster recover command to create
+From the rladmin CLI prompt, run the `cluster recover` command to create
 the new cluster based on the old cluster configuration. Note that this
 command is available in rladmin only if the node has not been configured
 yet.
 
 The command has the following parameters:
 
-1. filename - mandatory. The path of the old cluster configuration
+1. filename - mandatory. The full path of the old cluster configuration
     file. The file name is ccs-redis.rdb, and it is located in the /ccs
     directory of the persistent storage drive of the old cluster. This
     file exists on the persistent storage drive of all nodes and
     contains the same exact data, so it does not matter which old node's
     persistence storage you choose to use the file from.
+1. node_uid - mandatory.  The id of the first node, in this case 1.
```

Comment:
@Paz-Yanover Who needs to review this addition?

Reply:
@ushachar - who you want to review and approve it from your side?